packages/opentelemetry-node/docs/getting-started.mdx

```diff
@@ -37,7 +26,7 @@ configuration settings are `OTEL_EXPORTER_OTLP_*` to set the endpoint for
 sending data and `OTEL_SERVICE_NAME` to identify your service.
 
 The Distro will send telemetry data via OpenTelemetry's protocol (OTLP) to the
-configured endpoint (by default it sends to <http://localhost:4317>). The
+configured endpoint (by default it sends to `<http://localhost:4317>`). The
```

Comment:
```suggestion
configured endpoint (by default it sends to `http://localhost:4317`). The
```
If the goal is to not render that URL as a link, then perhaps lets drop the angle-brackets.

Reply:
Ohh, I didn't realize you wanted this to be a link. I can fix this by escaping the `<>` instead.

Reply:
```suggestion
configured endpoint (by default it sends to \< http://localhost:4317 \>). The
```